nvflare/app_opt/statistics/visualization/statistics_visualization.py

```diff
@@ -17,6 +17,17 @@
 from nvflare.fuel.utils.import_utils import optional_import
 
 
+def convert_data(feature_metrics) -> dict:
```

Comment:
If you had the "result collector" object, then this func could be a method of that object to export the data for different purposes.